examples/chapter-list.lua

```diff
@@ -1,3 +1,14 @@
+local settings = {
+    display_timeout = 5,
+  
+    loop_cursor = true,
+  
+    key_scroll_down = "DOWN WHEEL_DOWN",
+    key_scroll_up = "UP WHEEL_UP",
+    key_open_chapter = "ENTER MBTN_LEFT",
+    key_close_browser = "ESC MBTN_RIGHT",
+  }
+
```

Comment:
this should be underneath the header that explains how the script works

It should also use the mpv options API to allow users to easily change settings

Reply:
> this should be underneath the header that explains how the script works
> 
> It should also use the mpv options API to allow users to easily change settings

It should be fixed